ilc/client/pageTransitions.js

```diff
@@ -0,0 +1,59 @@
+let globalSpinner;
+const runGlobalSpinner = () => {
+    const tmplSpinner = window.ilcConfig && window.ilcConfig.tmplSpinner;
+    if (!tmplSpinner) return;
+
+    setTimeout(() => {
```

Comment:
I assume we should cancel this timeout if new app can be rendered faster than 200ms.